src/transformers/convert_pytorch_checkpoint_to_tf2.py

```diff
@@ -221,6 +223,12 @@
         LayoutLMForMaskedLM,
         LAYOUTLM_PRETRAINED_MODEL_ARCHIVE_LIST,
     ),
+    "layoutlmv2": (
+        LayoutLMConfig,
+        TFLayoutLMForMaskedLM,
+        LayoutLMForMaskedLM,
+        LAYOUTLM_PRETRAINED_MODEL_ARCHIVE_LIST,
+    ),
```

Comment:
Not sure if this PR needs to add LayoutLMv2 to the `convert_pytorch_checkpoint_to_tf2.py` script, as a TF implementation is not yet available.